app/views/admin/show.html.erb

```diff
@@ -7,21 +7,30 @@
   <p>Description: <%= @application.description %></p>
   <p>Status: <%= @application.status %></p>
   <p>Pets:</p>
-  <% @application.pets.each do |pet| %>
-    <%= link_to pet.name, "/pets/#{pet.id}" %>
-      <% end %>
+  </section>
+  
+  <div class="status">
   <% @application.application_pets.each do |app_pet| %>
     <% if app_pet.status == "Pending" %>
-    <%= form_with(url: "/admin/applications/#{@application.id}", data: {turbo: false}, method: :patch) do |form| %>
-        <%= form.hidden_field :app_pet_id, value: app_pet.pet_id %>
-        <%= form.submit "Approve" %>
-      <% end %>
+      <div>
+        <%= form_with(url: "/admin/applications/#{params[:id]}", data: { turbo: false }, method: :patch) do |form| %>
+          <%= form.hidden_field :app_pet_id, value: app_pet.id %>
+          <%= form.submit "Approve" %>
+        <% end %>
+        <%= form_with(url: "/admin/applications/#{params[:id]}", data: { turbo: false }, method: :patch) do |form| %>
+          <%= form.hidden_field :app_pet_id, value: app_pet.id %>
+          <%= form.submit "Reject" %>
```

Comment:
so we just needed to add a method to our applications model for reject, update the logic in our admin controller's update action, then update the html.erb with a conditional for rejected status rendering.  Great job on this!